src/Domain/DonationRepository.php

```diff
@@ -452,6 +452,30 @@ public function findWithTransferIdInArray(array $transferIds): array
         return $donations;
     }
 
+    /**
+     * @return Donation[]
+     */
+    public function findWithFeePossiblyOverchaged(): array
+    {
+        $query = $this->getEntityManager()->createQuery(<<<'DQL'
+            SELECT d
+            FROM MatchBot\Domain\Donation d 
+            WHERE d.collectedAt >= :start
+            AND d.donationStatus IN (:completeStatuses)
+            AND d.paymentMethodType = 'card'
+        DQL
+        );
+        $query->setParameter('start', new \DateTimeImmutable('2024-09-13 00:00:00'));
```

Comment:
Shouldn't this be September 12 instead of 13? The bug was introduced in https://github.com/thebiggive/matchbot/pull/1000 on Sept 6 and the first deploy to prod after that was https://github.com/thebiggive/matchbot/pull/990 on September 12.  Or we could just put it back to September 6th or even 1st, since checking a extra donations isn't a problem.

Reply:
Yes, looks like I inferred wrong from Jira.